src/components/10-atoms/link/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/link",
+  "version": "1.0.6",
```

Comment:
I didnt know we already did a prod release....

Reply:
next time please use alpha or beta tag till releasable

Reply:
IMHO it was releaseable. I thought that dist thing you guys were talking about was related to your table, sorry.

Reply:
We needed to define so its ok, but IMHO is not releasable if release part like dist are missing

Reply:
dont worry we are at the initial stage so is totally fine :)